ci/gpu/build.sh

```diff
@@ -55,29 +55,71 @@ gcc --version
 g++ --version
 conda list
 
-################################################################################
-# BUILD - Build and install librmm and rmm
-################################################################################
+if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
+    ################################################################################
+    # BUILD - Build and install librmm and rmm
+    ################################################################################
 
-logger "Build and install librmm and rmm..."
-"$WORKSPACE/build.sh" -v --ptds clean librmm rmm
+    logger "Build and install librmm and rmm..."
+    "$WORKSPACE/build.sh" -v --ptds clean librmm rmm
 
-################################################################################
-# Test - librmm
-################################################################################
+    ################################################################################
+    # Test - librmm
+    ################################################################################
+
+    if hasArg --skip-tests; then
+        logger "Skipping Tests..."
+    else
+        logger "Check GPU usage..."
+        nvidia-smi
 
-if hasArg --skip-tests; then
-    logger "Skipping Tests..."
+        logger "Running googletests..."
+
+        cd "${WORKSPACE}/build"
+        GTEST_OUTPUT="xml:${WORKSPACE}/test-results/" make -j${PARALLEL_LEVEL} test
+
+        logger "Python py.test for librmm_cffi..."
+        cd $WORKSPACE/python
+        py.test --cache-clear --junitxml=${WORKSPACE}/test-results/junit-rmm.xml -v --cov-config=.coveragerc --cov=rmm --cov-report=xml:${WORKSPACE}/python/rmm-coverage.xml --cov-report term
+    fi
 else
+    export LD_LIBRARY_PATH="$WORKSPACE/ci/artifacts/rmm/cpu/conda_work/build:$LD_LIBRARY_PATH"
+
+    TESTRESULTS_DIR=${WORKSPACE}/test-results
+    mkdir -p ${TESTRESULTS_DIR}
+    SUITEERROR=0
+
     logger "Check GPU usage..."
     nvidia-smi
 
     logger "Running googletests..."
+    # run gtests
+    cd $WORKSPACE/ci/artifacts/rmm/cpu/conda_work
+    for gt in "build/gtests/*" ; do
+        ${gt} --gtest_output=xml:${TESTRESULTS_DIR}/
+        exitcode=$?
+        if (( ${exitcode} != 0 )); then
+            SUITEERROR=${exitcode}
+            echo "FAILED: GTest ${gt}"
+        fi
+    done
 
-    cd "${WORKSPACE}/build"
-    GTEST_OUTPUT="xml:${WORKSPACE}/test-results/" make -j${PARALLEL_LEVEL} test
-
-    logger "Python py.test for librmm_cffi..."
     cd $WORKSPACE/python
+    
+    logger "Installing librmm..."
+    conda install -c $WORKSPACE/ci/artifacts/rmm/cpu/conda-bld/ librmm
```

Comment:
Would suggest using `--override-channels` or `--strict-channel-priority` here to ensure it doesn't fall back to finding `librmm` in the `rapidsai-nightly` or `rapidsai` channels

Reply:
I don't think either is necessary. Conda install first searches the `-c` channels, then `.condarc` channels. So we're guaranteed to find `librmm` in the local channel first. We need the `.condarc` channels from the container to get an dependencies, so I don't want to use `--override-channels`. And similarly, I don't want `--strict-channel-priority` to mess with the dependency resolution.